projects/go-lib/src/lib/components/go-toaster/go-toaster.component.html

```diff
@@ -1,12 +1,17 @@
 <!-- Here is where we make toast -->
-<div *ngIf="goToasterService.toasts" class="go-toaster">
-  <div @toastAnimation
-       *ngFor="let toast of goToasterService.toasts"
-       class="go-toaster__item">
-    <go-toast [header]='toast.header'
-              [message]="toast.message"
-              (mouseover)="pauseTimers()"
-              (mouseout)="restartTimers()"
-              [type]="toast.type"></go-toast>
+<div
+  *ngIf="goToasterService.toasts"
+  class="go-toaster">
+  <div
+    @toastAnimation
+    *ngFor="let toast of goToasterService.toasts"
+    class="go-toaster__item">
+    <go-toast
+      [header]='toast.header'
```

Comment:
I know this was just a formatting cleanup, but is there a reason to preserve the single quotes here?

Reply:
Nope, I didn't catch that when cleaning up this formatting. I fixed that, good catch!